### ISSUE TITLE
Replace typings with @types

### ISSUE THREAD

**Issue by domoritz** [microsoft/PowerBI-visuals#215]
The instructions in https://github.com/Microsoft/PowerBI-visuals/blob/master/Tutorial/Typings.md suggest the use of typings but withTypescript 2, [@types](https://basarat.gitbooks.io/typescript/docs/types/@types.html) is the [preferred way to get typings](https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/).

**Comment by ABKimmel** [comment]
You also don't have to add the typings manually to your `tsconfig.json` if you use the @types method.

**Comment by domoritz** [comment]
@ABKimmel, do you have an example project. I'm trying to get things working with @types but I'm getting a lot of weird errors.

**Comment by ABKimmel** [comment]
Not one I can share, unfortunately. I would recommend that you use `package.json` to manage your dependencies. So if I was going to add D3 to my project, I would add the line `"d3": ">1.0.0"` to the `dependencies` property of `package.json`. Then I would add a line for the typing as well: `"@types/d3": ">1.0.0.0"`. Then when you run `npm install` it's all handled for you. The TypeScript compiler should automatically find typings in the @types folder, according to the [documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types).

If you are willing to share your errors, I might be able to help.

**Comment by domoritz** [comment]
Thank you.

I added the dependencies to my `package.json`

```
"@types/d3": "^4.9.0",
"d3": "^4.9.1",
```

Then I added the path to d3 to my `pbiviz.json`.

```
  "externalJS": [
    "node_modules/powerbi-visuals-utils-dataviewutils/lib/index.js",
    "node_modules/d3/build/d3.js"
  ],
```

And lastly, I try to use d3 in my visual
```ts
console.log(d3)
```

But at that point d3 is undefined. This doesn't seem to be a problem with the typings as they shouldn't have any runtime impact. 

I also wonder why I cannot simply import d3 like

```
import * as d3 from 'd3';
```

PowerBI custom visuals seem to be making some assumptions about how the project is designed that I don't quite understand.
